whiteboard/helpers/blockBuilder.ts

```diff
@@ -110,16 +108,25 @@ export async function getSectionBlock(
     return block;
 }
 
-export async function getDividerBlock(): Promise<DividerBlock> {
+export function getMarkdownBlock(labelText: string) {
+    const block: SectionBlock = {
+        type: "section",
+        text: {
+            type: "mrkdwn",
```

Comment:
typo, should be `markown` isn't it?

Reply:
![image](https://github.com/RocketChat/Apps.Whiteboard/assets/92238941/b839a5aa-e937-4cf7-90e6-ad79e0450775)

I think signature type is mrkdwn only